indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -1,6 +1,31 @@
 {% extends 'events/surveys/management/_survey_base.html' %}
 
 {% block content %}
+    {% if survey.private and survey.state.name in ('active_and_clean', 'active_and_answered')%}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-lock"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}The survey is private{% endtrans %}
```

Comment:
In some places you use `{%-` and in others (like here) just `{%`. Choose your preferred one and use that :)